learn/wavemaker-known-issues.md

```diff
@@ -17,6 +17,7 @@ Below is the list of known issues reported against the release version. We docum
 
 |Issue description|Occured since|Expected to fix by|Status|Workaround|
 |---|---|---|---|---|
-|HQL grammar exception observed for queries with "Group by" clause when invoked using Java Service.|v10.6 | v10.6.5 <td bgcolor="FED788"> In-progress|-  |

```

Comment:
@neha2024, since the issue is fixed, can we mark this as fixed instead and remove the previously fixed item periodically?